guides/common/modules/proc_using-IdM-credentials-to-log-in-to-the-Project-Web-UI-with-a-Chrome-browser.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Using_IdM_credentials_to_log_in_to_the_{Project}_Web_UI-with-a-Chrome-browser_{context}"]
+= Using IdM credentials to log in to the {ProjectWebUI} with a Chrome browser
+
+This section describes how to use a Chrome browser to log in to your {ProjectWebUI} with your {FreeIPA} (IdM) login and password.
+
+.Prerequisites
+* Your system administrator has enrolled your {ProjectServer} into IdM and configured the server to use IdM for authentication.
+For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}#Using_FreeIPA_admin[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
```

Comment:
```suggestion
For more information, see {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}Using_FreeIPA_admin[Using {FreeIPA}] in _{InstallingServerDocTitle}_.
```

Reply:
Now that I look at the URL more closely, I believe it should be {InstallingServerDocURL}Configuring_External_User_Groups_{project-context}Using_FreeIPA_{project-context}[Using {FreeIPA}], should it not?

Reply:
The current URL: https://docs.theforeman.org/nightly/Installing_Server/index-katello.html#Using_FreeIPA_foreman

which becomes `{InstallingServerDocURL}` + `Using_FreeIPA_` + `{project-context}` (instead of "foreman") + `[Using FreeIPA]` (title copied from the rendered document) + `in _{InstallingServerDocTitle}_`.

Reply:
Does `{InstallingServerDocURL}Using_FreeIPA_{project-context}[Using FreeIPA]` also work for the Satellite downstream URL, namely https://access.redhat.com/documentation/en-us/red_hat_satellite/6.12/html/installing_satellite_server_in_a_connected_network_environment/configuring_external_authentication_satellite#Using_FreeIPA_satellite?
AFAICT, this ^^ is a link to the same section in the multiple-html format, but it seems differently constructed; there is an extra middle section, namely "configuring_external_authentication_satellite". Am I missing something or is it indeed weird that the upstream and downstream versions are of different lengths

Reply:
I have tried the following URL, and it is working for both satellite and katello:
{InstallingServerDocURL}Using_FreeIPA_{project-context}[Using {FreeIPA}]. So you were right in the original comment. (But it's also nice to have curly brackets around FreeIPA.)